pkgs/top-level/all-packages.nix

```diff
@@ -18401,10 +18398,7 @@ in
     pulseSupport = config.pulseaudio or false;
   };
 
-  velox = callPackage ../applications/window-managers/velox {
-    stConf = config.st.conf or null;
-    stPatches = config.st.patches or null;
-  };
+  way-cooler = callPackage ../applications/window-managers/way-cooler {};
```

Comment:
Any reason for moving `way-cooler` here?

Normally the attributes within categories should be sorted alphabetically, but especially this part of the file is pretty messed up anyway...

Reply:
@primeos No real reason, I've reverted that change